tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -91,28 +118,33 @@ static void *single_read_setup(void)
 
 ZTEST_USER(uart_async_single_read, test_single_read)
 {
-	uint8_t rx_buf[10] = {0};
-
-	/* Check also if sending from read only memory (e.g. flash) works. */
-	static const uint8_t tx_buf[5] = "test\n";
+	DATA_TYPE rx_buf[10] = {0};
+	const uint8_t nr_of_rx_values = sizeof(rx_buf)/sizeof(DATA_TYPE);
+#ifndef CONFIG_UART_WIDE_DATA
+	static const DATA_TYPE tx_buf[5] = "test\n";
+#else
+	static const DATA_TYPE tx_buf[5] = {0x0074, 0x0065, 0x0073, 0x0074, 0x000A};
+#endif
```

Comment:
I think you can just replace all of this by

static const DATA_TYPE tx_buf[5] = { 't', 'e', 's', 't', 0};

Reply:
Fixed